### PR TITLE
GRO-2284: reorganize package structure

### PR DESCRIPTION
**Pull request by KastorDriver**
[GRO-2284](https://selina.atlassian.net/browse/GRO-2284)

### **Refactoring of the package structure**
The current package structure is not intuitive and overloaded with many classes which don't belong to the same group.

We need that refactoring to make it intuitive and easier to understand where certain classes are stored.

The package structure should look like the structure of packages in the ms-authorization service.

The package reorganization should not cause any unexpected issues because the functionality of the application is not changing.

More details in the [Notion page](https://www.notion.so/selinaops/Refactor-the-Lending-API-packages-structure-and-write-proper-tests-for-controllers-and-HTTP-clients-761c5a7efef6408288d48e17e2d1afe2)

[GRO-2284]: https://selina.atlassian.net/browse/GRO-2284?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by selina-bot-user** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **kastordriver**
You can assign the PR to them by writing `/assign @kastordriver` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=Selina-Finance%2Fms-lending-api).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/Selina-Finance/ms-lending-api/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by selina-bot-user** [comment]
:star: PR built and available in a preview **jx-selina-finance-ms-lending-api-pr-199** [here](http://ms-lending-api-pr199.selinadev.co.uk)

**Comment by selina-bot-user** [comment]
:star: PR built and available in a preview **jx-selina-finance-ms-lending-api-pr-199** [here](http://ms-lending-api-pr199.selinadev.co.uk)

**Comment by selina-bot-user** [comment]
:star: PR built and available in a preview **jx-selina-finance-ms-lending-api-pr-199** [here](http://ms-lending-api-pr199.selinadev.co.uk)

**Comment by selina-bot-user** [comment]
:star: PR built and available in a preview **jx-selina-finance-ms-lending-api-pr-199** [here](http://ms-lending-api-pr199.selinadev.co.uk)

File: src/main/java/com/selina/lending/httpclient/middleware/dto/qqcf/request/QuickQuoteCFRequest.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+@Builder
+@Data
+public class QuickQuoteCFRequest {
+    String externalApplicationId;
+    String sourceAccount;
+    List<Applicant> applicants;
+    LoanInformation loanInformation;
+    PropertyDetails propertyDetails;
+    private Fees fees;
```
Comment (angelo-paiva-ptech) [inline src/main/java/com/selina/lending/httpclient/middleware/dto/qqcf/request/QuickQuoteCFRequest.java:20]: Any reason for fees to be the only private field?

**Review by angelo-paiva-ptech — APPROVED** [review]

**Comment by selina-bot-user** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *[angelo-paiva-ptech](https://github.com/Selina-Finance/ms-lending-api/pull/199#pullrequestreview-1525916431 "Approved")*

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=Selina-Finance%2Fms-lending-api).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/Selina-Finance/ms-lending-api/blob/main/OWNERS)~~ [angelo-paiva-ptech]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: src/main/java/com/selina/lending/httpclient/middleware/dto/qqcf/request/QuickQuoteCFRequest.java (hunk 1 above)
Comment (KastorDriver) [reply src/main/java/com/selina/lending/httpclient/middleware/dto/qqcf/request/QuickQuoteCFRequest.java:20]: Good catch!
Let me check all DTOs which are not @Value and make their properties private.

**Comment by selina-bot-user** [comment]
:star: PR built and available in a preview **jx-selina-finance-ms-lending-api-pr-199** [here](http://ms-lending-api-pr199.selinadev.co.uk)

**Comment by selina-bot-user** [comment]
:star: PR built and available in a preview **jx-selina-finance-ms-lending-api-pr-199** [here](http://ms-lending-api-pr199.selinadev.co.uk)

**Comment by selina-bot-user** [comment]
:star: PR built and available in a preview **jx-selina-finance-ms-lending-api-pr-199** [here](http://ms-lending-api-pr199.selinadev.co.uk)

File: src/main/java/com/selina/lending/httpclient/middleware/dto/qqcf/request/QuickQuoteCFRequest.java (hunk 1 above)
Comment (KastorDriver) [reply src/main/java/com/selina/lending/httpclient/middleware/dto/qqcf/request/QuickQuoteCFRequest.java:20]: Fixed!

**Comment by selina-bot-user** [comment]
:star: PR built and available in a preview **jx-selina-finance-ms-lending-api-pr-199** [here](http://ms-lending-api-pr199.selinadev.co.uk)

**Pull request merged**